scripts/create_whitelist.sh

```diff
@@ -0,0 +1,108 @@
+#!/bin/bash
```

Comment:
I wonder why this file is not moved from keylime/
I thought move this to scripts/ too, but it seems some scripts depending on it, some are used by CI, some are used by other scripts that I don't have history background on.
If the rename is a two phase step, then it's OK.

Reply:
This is just a useful script to generate whitelists for IMA. It unarchives an initamfs and iterates over each file with a sha checksum. It only has a reference in the demo scripts.

Reply:
Ah, I remember I used this script in the evaluation, I guess there is no way to get the the checksum of boot aggregate? It looks like a runtime output, please forgive my limited knowledge on the IMA.

Reply:
Do you mean get it before first boot? 

It might be the same between identical machines and OSes.

Reply:
unless this might be of help:

http://linux-ima.sourceforge.net/linux-ima-measurements.html

Reply:
Thanks for the link, noted. Yep, so the trust solutions as I am aware need profiling a trust node for the target state.